src/lib/efrm/efrm_efct_rxq.c

```diff
@@ -189,10 +189,14 @@ int efrm_rxq_refresh(struct efrm_efct_rxq *rxq, unsigned long superbufs,
 	struct xlnx_efct_hugepage spare_page = {};
 	size_t i;
 	int rc = 0;
+	unsigned n_hugepages = CI_MIN(max_superbufs, (unsigned)CI_EFCT_MAX_SUPERBUFS) /
+                         CI_EFCT_SUPERBUFS_PER_PAGE;
 
 	if (max_superbufs != CI_EFCT_MAX_SUPERBUFS) {
-		/* Could be supported without much difficulty, but no need for now */
-		return -EINVAL;
+		EFRM_TRACE("max_superbufs: %u passed in by user not equal to kernel's: %u. "
```

Comment:
I think I'd like to be noisier in the event that it's definitely going to go wrong, i.e. when `pages` has a non-NULL entry beyond max_superbufs then log an error. The goal is to make such failures in the field more debuggable.